library/src/main/java/com/novoda/downloadmanager/DownloadBatch.java

```diff
@@ -85,10 +89,6 @@ void download() {
             DownloadsNetworkRecoveryCreator.getInstance().scheduleRecovery();
         }
 
-        if (status == DOWNLOADED) {
```

Comment:
This was always false because of the guard at the beginning of this class 😬  so all batches were marked as `DOWNLOADING` even when they were `DOWNLOADED` 😬